src/App.js

```diff
@@ -52,7 +69,7 @@ class App extends Component {
     let price
 
     DATA.forEach((dataItem) => {
-      if (dataItem.id === parseInt(event.target.id)) {
+      if (dataItem.id === parseInt(event.target.id, 10)) {
         price = dataItem.price
       }
     })
```

Comment:
I hope we will have time to will learn about more functional techniques. For example, `find`. You can write this function much more efficiently like this:
```javascript
const targetId = parseInt(event.target.id, 10)
const price = DATA.find((dataItem) -> {
  return dataItem.id === targetId
})
```